Gemfile

```diff
@@ -27,13 +27,13 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 gem 'envyable'
 
 # For interacting with the ActionKit API.
-gem 'actionkit_connector'
+gem 'actionkit_connector', '~> 0.1.1'
 
 # For pushing out updates after processing an event.
 gem 'pusher'
 
 # For processing messages from SQS
-gem 'shoryuken'
+# gem 'shoryuken'
```

Comment:
`shoryuken` is relying on a discontinued version of `celluloid` (0.16.1), so I'm removing the requirement at the moment since it isn't required for the correct operation of this system.